_includes/features.html

```diff
@@ -57,12 +57,7 @@
                 <p class="second-title"><i class="fas fa-chart-pie"></i> &nbsp; Analytics
                 </p>
                 <div>
-                    <p class="feature-text">JanusGraph global graph data analytics integrations include:</p>
-                    <ul class="central-list">
-                        <li><a class="feature-link" href="https://spark.apache.org">Apache Spark</a></li>
-                        <li><a class="feature-link" href="https://hadoop.apache.org">Apache Hadoop</a></li>
-                        <li><a class="feature-link" href="https://giraph.apache.org">Apache Giraph</a></li>
-                    </ul>
+                    <p class="feature-text">JanusGraph supports global graph analytics with <a class="feature-link" href="https://spark.apache.org">Apache Spark</a>.</p>
```

Comment:
Looks good but I think maybe it is better to make this paragraph just a little bit bigger so that it would be similar to other `feature` paragraphs in size?
I am not sure, but maybe it would be better to change it to something like:
`In addition to online transactional processing (<a class="feature-link" href="https://en.wikipedia.org/wiki/Online_transaction_processing">OLTP</a>), JanusGraph supports global graph analytics (<a class="feature-link" href="https://en.wikipedia.org/wiki/Online_analytical_processing">OLAP</a>) with the native <a class="feature-link" href="https://spark.apache.org">Apache Spark</a> integration.`

Reply:
Sounds good in general, but what does "_native_ Apache Spark integration" mean? I would just write "with Apache Spark" or maybe "with its Apache Spark integration".

Reply:
You are right. Better without `native`

Reply:
OK, I changed that. Preview can be found [here](https://florianhockmann.github.io/janusgraph.org/).